### DISCUSSION TITLE
Find what schedule is running from inside an op

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#8416] [Q&A] [answered]
Is there a way to access to what schedule is running from inside an op of a run from Python?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1642176334390800?thread_ts=1642176334.390800&cid=C01U954MEER).*

**Comment by yuhan** [comment, marked as answer]
You can find the information as:
```
@op
def my_op(context):
    context.get_tag("schedule_name")
```

If you'd like to get other information, you can pass it from your schedule definition via tags. You can find examples in the [docs](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules#a-schedule-that-provides-custom-run-config-and-tags)
